etc/profile-a-l/blink-common-hardened.inc.profile

```diff
@@ -0,0 +1,10 @@
+# Firejail profile alias for chromium-common-hardened.inc
+# This file is overwritten after every install/update
+# Persistent local customizations
+include blink-common-hardened.inc.local
+# Persistent global definitions
+# added by caller profile
+#include globals.local
+
+# Redirect
+include chromium-common-hardened.inc.profile
```

Comment:
This seems backwards, as chromium-common.profile redirects to
blink-common.profile:

* chromium-common.profile -> blink-common.profile
* blink-common-hardened.inc.profile -> chromium-common-hardened.inc.profile

I think it would be better to remove this file.

Alternatively, test Blink-based programs with
chromium-common-hardened.inc.profile and if that works:

* Move the entries from chromium-common-hardened.inc.profile to this file
* Make chromium-common-hardened.inc.profile redirect to this file instead

@rusty-snake Since you worked on #3688, thoughts on this?